crates/stc_ts_type_checker/tests/conformance/controlFlow/controlFlowIfStatement.ts.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
     required_error: 0,
     matched_error: 0,
-    extra_error: 0,
+    extra_error: 1,
```

Comment:
Can you investigate this?

Reply:
Yeah I added an explanation in the description as to why it's increased